linera-storage/src/db_storage.rs

```diff
@@ -855,7 +848,7 @@ where
             #[cfg(with_metrics)]
             metrics::READ_CERTIFICATES_COUNTER
                 .with_label_values(&[])
-                .inc();
+                .inc_by(hashes.len() as u64);
```

Comment:
(So now it's the number of certificates, rather than the number of `read_certificates` calls?)